libnavigation-core/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,10 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="com.mapbox.navigation.core">
 
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
We were bringing those from upstream libraries (Events, Maps and others). Great catch 👍 

Refs. https://github.com/mapbox/mobile-telemetry/issues/445